src/aat/resources/features/F-027/S-035.td.json

```diff
@@ -0,0 +1,52 @@
+{
+  "_guid_": "S-035",
+  "_extends_": "S-000-Base-Root",
+  "title": "must return appropriate negative response when case-reference does not exists",
+  "specs": ["a detailed profile in CCD",
+    "does not provide the valid case-reference in input parameters",
+    "has the 400 return code"],
+  "productName": "CCD Data Store",
+  "operationName": "Fetch a case for display for Case Worker",
+  "request": {
+    "headers": {
+      "Authorization": "[[DYNAMIC]]"
+    },
+    "pathVariables": {
+      "uid": "[[DYNAMIC]]",
+      "jid": "AUTOTEST1",
+      "ctid": "AAT",
+      "cid": "DummyReferenceNumber"
+    }
+  },
+  "method": "GET",
+  "uri": "/aggregated/caseworkers/{uid}/jurisdictions/{jid}/case-types/{ctid}/cases/{cid}",
+  "expectedResponse": {
+    "responseCode" : 400,
+    "responseMessage" : "Bad Request",
+    "headers" : {
+      "X-Frame-Options" : "DENY",
+      "Transfer-Encoding" : "chunked",
+      "Cache-Control" : "no-cache, no-store, max-age=0, must-revalidate",
+      "vary" : "accept-encoding",
+      "X-Content-Type-Options" : "nosniff",
+      "Content-Encoding" : "gzip",
+      "Connection" : "close",
+      "Pragma" : "no-cache",
+      "Expires" : "0",
+      "X-XSS-Protection" : "1; mode=block",
+      "Date" : "[[ANY]]",
+      "Content-Type" : "application/json;charset=UTF-8"
```

Comment:
These should be inherited from base object

Reply:
Changes done and updated.